### ISSUE TITLE
Release 1.2.2 to Maven Central

### ISSUE THREAD

**Issue by spikymonkey** [spring-cloud/spring-cloud-app-broker#433]

**Comment by abelsromero** [comment]
Note that 1.2.2 is in libs-release, but not in central.

**Comment by spikymonkey** [comment]
I gave this a go today but the job got stuck due to not having enough version history for the `artifactory-repo` resource. I forced a fetch of some more history with `fly -t app-broker check-resource -r app-broker-1.2.x/artifactory-repo --from build_number:app-broker-1.2.x-build-6` which allowed the job to trigger, but then it failed with 

```
Caused by: java.lang.IllegalArgumentException: URI is not absolute
	at java.net.URI.toURL(URI.java:1088) ~[na:1.8.0_292]
	at org.springframework.http.client.SimpleClientHttpRequestFactory.createRequest(SimpleClientHttpRequestFactory.java:145) ~[spring-web-5.3.8.jar!/:5.3.8]
```

I don't really understand the design of the pipeline which makes me worry about triggering this job. It's pulling `build_number	app-broker-1.2.x-stage-release-2` from Artifactory - shouldn't we be pulling a promoted release? I assume this _is_ the version that was promoted, but still it feels weird. And also the fact that it seems to pull down `1.2.0` and `1.2.2` artifacts does not fill me with the kind of confidence I'd like to have when hitting the big 'make this all public' button...
